src/main/java/edu/hm/hafner/analysis/registry/DoxygenDescriptor.java

```diff
@@ -23,6 +23,8 @@ public IssueParser createParser(final Option... options) {
 
     @Override
     public String getHelp() {
-        return "Use commandline <code>( cat Doxyfile; echo WARN_FORMAT='$file:$line: $text' ) | doxygen -</code>";
+        return "Execute doxygen:"
+              + "As <b>shell</b> command <code>( cat Doxyfile; echo WARN_FORMAT='$file:$line: $text' ) | doxygen -</code>"
+              + "As <b>batch</b> command <code>( type Doxyfile & echo WARN_FORMAT='$file:$line: $text' ) | doxygen -</code>";
```

Comment:
POSIX shells remove the single-quotation marks from the `echo` command, but the Windows `cmd` command interpreter doesn't. Can that difference cause any problem?